template_app/src/charts/ApiBarChart.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  // Example from: https://www.chartjs.org/docs/latest/getting-started/usage.html
+  import { Chart } from "chart.js/auto";
+  import { getAquisitionsByYear } from "./api";
+  import { onMount } from "svelte";
+  export let title: string;
```

Comment:
Style nit: I usually see a newline separating the imports from the code, and between `let` declarations and any functions

Reply:
Yes agreed, added one now.